virttest/qemu_vm.py

```diff
@@ -1368,9 +1368,6 @@ def sort_key(dev):
         if root_dir is None:
             root_dir = self.root_dir
 
-        have_ahci = False
-        have_virtio_scsi = False
-        virtio_scsi_pcis = []
```

Comment:
@hereischen     What is this change for ?  From my understanding, it is not related with "destination vm does not be destroyed", right ?

Reply:
just delete some vars that were not used